graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx

```diff
@@ -18,12 +18,12 @@ class BootstrapModalForm extends Component {
     this.open = this.open.bind(this);
     this.close = this.close.bind(this);
     this._submit = this._submit.bind(this);
-    this._onModalClose = this._onModalClose.bind(this);
+    this._onModalCancel = this._onModalCancel.bind(this);
   }
 
-  _onModalClose() {
-    if (typeof this.props.onModalClose === 'function') {
-      this.props.onModalClose();
+  _onModalCancel() {
+    if (typeof this.props.onCancel === 'function') {
```

Comment:
I don't really see it the same way, and also don't consider that a user making the modal go away, is implying that their changes are gone. We are imprecise by nature, and will find the easiest way to make that go away, even if you just want to look at some detail or copy something from somewhere else.

Either way, I don't think we can convince each other on this, and I don't want to discuss this forever. As long as the changes work consistently, we can go with them.